cct/module.py

```diff
@@ -374,6 +375,8 @@ def _run_function(self, name, *args):
             env = dict(os.environ)
             env['CCT_MODULE_PATH'] = os.path.dirname(self.script)
             for name, res in self.cct_resource.items():
+                var_name = 'CCT_ARTIFACT_PATH_' + name.upper()
+                logger.info('Created %s artifact' % var_name)
```

Comment:
This needs to be cleared. It's not artifact, but artifact env variable that is a path to artifact. Change it to:

`Created XYZ environment variable with path to FILE_PATH artifact`